Introduction.md

```diff
@@ -299,34 +299,10 @@ Running `git-webkit setup` configures your git hooks to properly generate commit
 The first line contains the date, your full name, and your email address.
 Use this to write up a brief summary of the changes you’ve made.
 Don’t worry about the “Reviewed by NOBODY (OOPS!)” line, the person landing your patch will fill this in.
-There is one ChangeLog per top-level directory.
 
-A typical change log entry before being submitted to [bugs.webkit.org](https://bugs.webkit.org/) looks like this:
```

Comment:
Seems like we should provide a commit message example here.

Reply:
Sure, I can add that